src/store/index.js

```diff
@@ -0,0 +1,4 @@
+import { createStore } from 'redux';
+import { contactReducer } from './contactReducer';
+
+export const store = createStore(contactReducer);
```

Comment:
create many reducers . use [combineReducers](https://redux.js.org/api/combinereducers) api.
for now we will have only one, but its clear and useful to have many